web/src/main.js

```diff
@@ -54,11 +54,12 @@ import Delete from '@/pages/Delete.vue'
 
 const routes = [
   { path: '/', component: Home },
-  { path: '/:project/:version?/:location(.*)?', component: Docs },
+  { path: '/:project/:version/:location(.*)?', component: Docs },
   { path: '/help', component: Help },
   { path: '/upload', component: Upload },
   { path: '/claim', component: Claim },
   { path: '/delete', component: Delete },
+  { path: '/:project', redirect: '/:project/latest' },
```

Comment:
You may want to revert this change now that the fix for #137 has been merged. Furthermore, the `latest` tag is nothing which is currently really "known" or enforced by the logic. You still have to create the `latest` tag manually with e.g. : `curl -X PUT http://localhost:8000/api/PROJECT/VERSION/tags/latest`